### DISCUSSION TITLE
I am experiencing a date-time error when trying to run simulations using Alpyne.

### DISCUSSION THREAD

**Discussion by pcsksa5** [the-anylogic-company/Alpyne#60] [Q&A]
I am experiencing an error when trying to run simulations using Alpyne. The error message is:
ValueError: ':' is a bad directive in format '%Y-%m-%dT%H:%M:%S.%f%

This occurs when initializing the simulation with various `.jar` files, including both the Pathfinder and Stock Management models.

**Steps Taken:**
- Verified Java 17 installation and configuration.
- Updated Alpyne to the latest version.
- Checked and corrected the `PATH` environment variable.
- Restarted the system and re-tested.
- Tried different models and ensured correct date-time formats.

**Environment:**
- Windows 10
- Python 3.12
- anylogic-alpyne version 1.2.0
- AnyLogic version 8.9.0

**Request:**
Could someone provide guidance on resolving this date-time format issue? Any suggestions for debugging or potential fixes would be greatly appreciated.

Thank you!

**Comment by oreoru** [comment]
delete":" in "%:z" from "%Y-%m-%dT%H:%M:%S.%f:%z",which in constants.py line 13 and try to run again
